### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AgggR**

### CODE REVIEW

**Comment by AgggR** [comment]
[AR_LaTeX homework (1).zip](https://github.com/nlp-ug-2018-19/LaTeX-homework/files/3269809/AR_LaTeX.homework.1.zip)
[AR_LaTeX_homework.pdf](https://github.com/nlp-ug-2018-19/LaTeX-homework/files/3269810/AR_LaTeX_homework.pdf)

File: AR_LaTeX_homework.tex [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+\section{Section 1}
+\begin{enumerate}
+  \item Neurology (brain possibilities).
+  \item Phonology in general.
+  \item Somnology (about dreaming in general, includes lucid dreaming).
+\end{enumerate}
+\section{Section 2}
+
+% aɪm ən ˈɔːl.məʊst ˈtwɛn.ti-faɪv jɪr əʊld ˈstjuː.dənt ɒv ˈɪŋ.ɡlɪʃ fɪˈlɒl.ə.dʒi | maɪˌspeʃ.əl.aɪˈzeɪ.ʃən  ɪz ˈnætʃ.ər.əl  ˈlæŋ.ɡwɪdʒ ˈprəʊ.ses.ɪŋ  |   aɪm ɪn ðə ˈmɪdl ɒv maɪ ˌsek.ənd.dɪˈɡriː ænd maɪ ˈɪn.trəsts   ɑː  ˈlɜː.nɪŋ ˌdʒæp.ənˈiːz |  dɑːnsing   dʒæz  and   ˈfɪɡ.ə ˌskeɪ.tɪŋ wɪtʃ  ɪnˈkluːdz on ˈaɪs ˌskeɪ.tɪŋ | ˈfɪɡ.ə  ˈrəʊ.lə.skeɪ.tɪŋ and off-ice ˈek.sə.saɪzIz/
```
Comment (MarcinFortuna-UG) [inline AR_LaTeX_homework.tex:28]: Is it some online transcription engine? :)
Please note that it syllabifies in a way which does not observe Maximal Onset Principle.

File: AR_LaTeX_homework.tex [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+%Moimi zainteresowaniami są nauka języka japońskiego, tańczenie dżezu oraz jazda figurowa, która obejmuje jazdę na łyżwach, jazdę na łyżworolkach figurowych oraz imitacje czyli ćwiczenia poza lodem.
+\newline
+\newline
+\textipa{/jEstEm dvu\textdctzlig est\textopeno \texttoptiebar{\textteshlig}ter\textopeno let\textltailn  \~\textopeno \space studentk\~\textopeno \space fil\textopeno l\textopeno g\textsuperscript{j}i \~a\ng g\textsuperscript{j}Elsk\textsuperscript{j}ej | na spEcjal\textsuperscript{j}izac\textsuperscript{j}i pSEtfa\textyogh an\textsuperscript{j}e j\~E\~wz\textbari ka naturalnEg\textopeno \space || m\textopeno \textsuperscript{j}imi zaintErEs\textopeno van\textsuperscript{j}ami s\~\textopeno \space | nauka j\~Ez\textbari ka naturalnEg\textopeno \space | t\~a\~j\texttoptiebar{\textteshlig}En\textsuperscript{j}E d\texttoptiebar{\textyogh}Ezu \textopeno raz jazda figur\textopeno va | ktura obEjmujE jazdE na w\textbari\textyogh vax | jazdE na w\textbari\textyogh v\textopeno r\textopeno lkax figur\textopeno v1x | \textopeno raz imitacjE | \texttoptiebar{\textteshlig}1li t\texttoptiebar{\textctc}fi\textteshlig En\textsuperscript{j}a p\textopeno za l\textopeno dEm/}
```
Comment (MarcinFortuna-UG) [inline AR_LaTeX_homework.tex:39]: "nauka języka naturalnego"?
Why no word stress here?
Polish <c> = /ts/ (not /c/)!

**Review by MarcinFortuna-UG — APPROVED** [review]
Dear Agnieszka, 

the homework looks good, I'm approving and merging in this form. However, you may still want to correct some minor things. Please read the inline comments below.

**Pull request merged**